harness/src/test/resources/pvt.yaml

```diff
@@ -7,8 +7,15 @@ product:
   version: 1.0
   milestone: CR1
 sourceDistribution:
-  #foo.zip
-distributionDirectory:
-  #/tmp/foo.zip
+  /jboss-eap-7.0.0.CR2-src.zip
+distribution:
+  /jboss-eap-7.0.0.CR2.zip
 mavenRepository:
-  #/tmp/foo.zip
+  /jboss-eap-7.0.0.CR2-maven-repository.zip
+testCases:
```

Comment:
As the current suggestion from @ryanzhang is limited manual testing and some unit testing please remove these.